internal/controller/dscinitialization/dscinitialization_controller.go

```diff
@@ -452,6 +452,15 @@ func (r *DSCInitializationReconciler) newMonitoringCR(ctx context.Context, dsci
 
 	defaultMonitoring.Spec.Traces = dsci.Spec.Monitoring.Traces
 
+	// if metrics or traces is set, we set collector replicas to the value set in the DSCInitialization resource
+	if dsci.Spec.Monitoring.Metrics != nil || dsci.Spec.Monitoring.Traces != nil {
```

Comment:
Should this be modified to account for single node clusters also? or should 2 be the default for both?

Reply:
Not yet, I think there's a separate issue for figuring out how to identify we are in a single node env and set replicas accordingly.